modules/460-log-shipper/crds/cluster-log-destination.yaml

```diff
@@ -220,18 +220,19 @@ spec:
                       description: The base URL of the Elasticsearch instance.
                     dataStreamEnabled:
                       type: boolean
-                      default: true
+                      default: false
                       description: |
                         Use for storage indexes or datastreams (https://www.elastic.co/guide/en/elasticsearch/reference/master/data-streams.html).
 
-                        Datastream usage is better as logs and metrics storage.
-                    docTypeSupport:
-                      type: boolean
-                      default: false
+                        Datastream usage is better for logs and metrics storage but they works only for Elasticsearch >= 7.16.X.
+                    docType:
+                      type: string
```

Comment:
Could you elaborate more on why do we need string type here? As for me, enum with three options seems more preferable.

Reply:
Users can have their own index created and configured with their custom doc_type. We can't predict every value for the doc_type. Also, this field is needed only by 5.X ES

Reply:
My concern is if I have an instance of 5.x ElasticSearch, how do I know which docType should I set? Can we provide some commands to find it? If not, it is ok.

Reply:
If you (as a client) don't mind. You can set 'vector' value. I left it as an example in README and CRD.
If you have some special case (for example you have created an index with your own doc_type, then I think you know, what you have set here).